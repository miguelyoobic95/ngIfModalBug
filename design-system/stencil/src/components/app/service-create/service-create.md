---
name: ServiceCreate
category: Core Components
---

```service-create.html
    <service-create></service-create>
```

## Attributes

|Name|Type|Default|Description|
|---|---|---|---|
|`text`|string|   |the text inside the label|

## CSS

|Type|Name|Description|
|---|---|---|
|Accent|`accent`|service-create accent|
|Dark|`dark`|service-create dark|
|Success|`success`|service-create success|
|Danger|`danger`|service-create danger|
|Info|`info`|service-create info|
|Warning|`warning`|service-create warning|
|Gradient Accent|`gradient-accent`|service-create gradient-accent|
|Gradient Dark|`gradient-dark`|service-create gradient-grey|
|Gradient Success|`gradient-success`|service-create gradient-success|
|Gradient Danger|`gradient-danger`|service-create gradient-danger|
|Gradient Info|`gradient-info`|service-create gradient-info|
|Gradient Warning|`gradient-warning`|service-create gradient-warning|

## Events
|Attr|Description|
|---|---|---|
|`yooXXXX`|small label|