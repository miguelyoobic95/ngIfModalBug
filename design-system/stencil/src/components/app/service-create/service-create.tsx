import { Component, Element } from '@stencil/core';

@Component({
    tag: 'yoo-service-create',
    styleUrl: 'service-create.scss',
    scoped: true
})
export class YooServiceCreateComponent {

     @Element() host: HTMLStencilElement;

    render(): JSX.Element {
        return (
            <div>ServiceCreate needs a proper template</div>
        );
    }
}
