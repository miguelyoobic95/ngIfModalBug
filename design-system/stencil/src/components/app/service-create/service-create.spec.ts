import { TestWindow } from '@stencil/core/testing';
import { YooServiceCreateComponent } from './service-create';

describe('ServiceCreateComponent', () => {
    it('should build', () => {
        expect(new YooServiceCreateComponent()).toBeTruthy();
    });

    describe('Rendering', () => {
        let element, window;
        beforeEach( async () => {
            window = new TestWindow();
            element = await window.load({
                components: [YooServiceCreateComponent],
                html: '<yoo-service-create></yoo-service-create>'
            });
        });

        it ('Should render', async () => {
            expect(element).toMatchSnapshot();
        });
    });

    describe('Events', () => {

        let emitEvent, window;
        beforeEach( async () => {
            window = new TestWindow();
            emitEvent = jest.fn();
        });
    });
});