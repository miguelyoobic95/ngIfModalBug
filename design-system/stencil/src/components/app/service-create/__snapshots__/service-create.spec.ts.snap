// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServiceCreateComponent Rendering Should render 1`] = `
<yoo-service-create
  class="hydrated"
  data-yoo-service-create-host=""
>
  
  <div
    data-yoo-service-create=""
  >
    ServiceCreate needs a proper template
  </div>
</yoo-service-create>
`;
